tests/test_h5store.py

```diff
@@ -703,9 +709,9 @@ def test_single_writer_multiple_reader_different_process_no_swmr(self):
     @unittest.skipUnless(python_implementation() == 'CPython', 'SWMR mode not available.')
     def test_single_writer_multiple_reader_different_process_swmr(self):
 
-        read_cmd = "python -c 'from signac.core.h5store import H5Store; "
```

Comment:
Is there a reason for these changes other than aesthetics? I don't see any difference in windows compatibility.

Reply:
It's a compatibility fix for how Windows' `cmd` handles single quotes. It's different than bash. I just picked a syntax that I knew would work for both.

Reply:
Ah, I wouldn't have thought of that.